sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryIO.java

```diff
@@ -402,7 +404,10 @@ public static Read read() {
    * domain-specific type, due to the overhead of converting the rows to {@link TableRow}.
    */
   public static TypedRead<TableRow> readTableRows() {
```

Comment:
In this case I think we should add another readTableRowsWithSchema method, if only because this source is widely use and I want to be conservative about adding schemas to the main code path.